catalog/transformer/catalog-transformer-pdf/src/main/java/ddf/catalog/transformer/input/pdf/PdfInputTransformer.java

```diff
@@ -64,15 +67,24 @@ public Metacard transform(InputStream input, String id)
             throws IOException, CatalogTransformerException {
         try (PDDocument pdfDocument = PDDocument.load(input)) {
             return transformPdf(id, pdfDocument);
+        } catch (InvalidPasswordException e) {
+            LOGGER.warn("Cannot transform encrypted pdf", e);
+            return initializeMetacard(id);
         }
     }
 
-    private Metacard transformPdf(String id, PDDocument pdfDocument) throws IOException {
+    private MetacardImpl initializeMetacard(String id) {
         MetacardImpl metacard = new MetacardImpl();
 
         metacard.setId(id);
         metacard.setContentTypeName(MediaType.PDF.subtype());
```

Comment:
For some reason the application part gets added on somewhere else. So I put it as just PDF